### ISSUE TITLE
Creating 'greta-env' conda environment using python v3.7 failed

### ISSUE THREAD

**Issue by kris1yi** [greta-dev/greta#573]
Please briefly describe your problem and what output you expect.

If you have a question, please don't use this form. Instead, ask on <https://forum.greta-stats.org/>.

Please include a minimal reproducible example (AKA a reprex). If you've never heard of a [reprex](http://reprex.tidyverse.org/) before, start by reading <https://www.tidyverse.org/help/#reprex>.

---
```r
`greta::install_greta_deps()`
```
greta::install_greta_deps()
✖ Creating 'greta-env' conda environment using python v3.7, this may take a minute ... failed
+ '/cloud/home/r1747991/.local/share/r-miniconda/bin/conda' 'create' '--yes' '--name' 'greta-env' 'python=3.7' '--quiet' '-c' 'conda-forge'

have tried these in R cloud
```
```r
reticulate::install_miniconda(force = TRUE)
reticulate::conda_create(envname = 'greta-env', python_version = '3.7')
reticulate::conda_install(envname = 'greta-env', 
                          packages = c('numpy==1.16.4',
                                       'tensorflow-probability==0.7.0', 
                                       'tensorflow==1.14.0'))

```
```
CondaValueError: prefix already exists: /cloud/home/r1747991/.local/share/r-miniconda/envs/greta-env
Error: Error creating conda environment 'greta-env' [exit code 1]

**Comment by njtierney** [comment]
Hi there,

You are running this on a cloud service, is that correct? In the past we have found that we needed to increase RAM to about 4Gb in order to install dependencies - we should probably mention this in the help message that follows, as this comes up a bit.

Are you using 4Gb of RAM on your cloud service? If not, then that would be great to be able to run it with more RAM?

Let me know how you go!
